assets/app/view/map_page.rb

```diff
@@ -12,6 +12,7 @@ class MapPage < Snabberb::Component
 
     def render
       game_title = @route.match(ROUTE_FORMAT)[1]
+      game_title = game_title[0..-2] if game_title[-1] == '/'
```

Comment:
can't this be done with regex?

    ROUTE_FORMAT = %r{/map/(.*)/?}.freeze

Reply:
2b84872